src/js/view/controls/settings-menu.js

```diff
@@ -133,9 +133,23 @@ export function setupSubmenuListeners(settingsMenu, controlbar, viewModel, api)
         );
     };
 
+    const changeAutoLabel = function (quality, qualitySubMenu, currentQuality) {
+        const items = qualitySubMenu.getItems();
+        const item = items[0].element().querySelector('.jw-auto-label');
+        const levels = model.get('levels');
+        const { level, mode } = quality;
+
+        item.textContent = currentQuality || mode === 'manual' ? '' : levels[level.index].label;
```

Comment:
When would `currentQuality` be greater than `0` and the player not be in manual mode? The last visual quality event could be a manual change, but that doesn't matter if the user selected a quality, unless we want to continue showing the auto quality label until the quality actually changes.

Reply:
I think we should remove `|| mode === 'manual'`  - that just prevents the label from being added until an ABR switch. If you were in say "240p" and then picked "auto", we should just immediately see "Auto 240p" because we know the visual quality, and it does matter that we got there in manual mode.